src/main/java/reactor/netty/transport/ProxyProvider.java

```diff
@@ -238,6 +248,7 @@ private String getPasswordValue() {
 		String nonProxyHosts;
 		Supplier<? extends HttpHeaders> httpHeaders;
 		Proxy type;
+		long connectTimeoutMillis;
```

Comment:
Please set a default value
https://github.com/netty/netty/blob/662e0b4b81833a835b290e60982c9de3a1878656/handler-proxy/src/main/java/io/netty/handler/proxy/ProxyHandler.java#L46

Otherwise with this change you effectively disable the default implementation in `ProxyHandler`